app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -22,6 +22,7 @@ function MiddlewareTopologyCtrl($scope, $http, $interval, $location, topologySer
             $scope.items = data.data.items;
             $scope.relations = data.data.relations;
             $scope.kinds = data.data.kinds;
+            $scope.icons = data.data.icons;
```

Comment:
have you checked containers topology impl? iirc I didn't use scope there and was able to test the icons